### PR TITLE
Added initial documentation for models

### PR DESCRIPTION
**Pull request by kriscon-db**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
Initial version of model documentation.

Quickstart manually verified on my local machine.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -146,6 +146,84 @@ Delete the table to clean up:
 bin/uc table delete --full_name unity.default.my_table
 ```
 
+## Manage models in Unity Catalog using MLflow
+
+Unity catalog supports the management and governance of ML models as securable assets.  Starting with
```
Comment (dennyglee) [inline docs/quickstart.md:151]: Could you please capitalize "Catalog"?

**Review by dennyglee — DISMISSED** [review]

File: docs/quickstart.md [hunk 2]
```diff
@@ -146,6 +146,84 @@ Delete the table to clean up:
...
+[MLflow 2.16.1](https://mlflow.org/releases/2.16.1), MLflow offers integrated support for using Unity Catalog as the 
+backing resource for the MLflow model registry.  What this means is that with the MLflow client, you will be able to 
+interact directly with your Unity Catalog service for the creation and access of registered models.
+
+## Setup MLflow for usage with Unity Catalog
+
+In your desired development environment, install MLflow 2.16.1 or higher:
+
+```sh
+$ pip install mlflow>=2.16.1
```
Comment (kriscon-db) [inline docs/quickstart.md:161]: This should just be pip install mlflow now that 2.16.1 is published.  Will update.

File: docs/quickstart.md [hunk 3]
```diff
@@ -146,6 +146,84 @@ Delete the table to clean up:
...
+[2023-10-25 19:39:12 -0700] [50239] [INFO] Listening at: http://127.0.0.1:5000 (50239)
+```
+
+Next, from within a python script or shell, import MLflow and set the tracking URI and the registry URI.
+
+```python
+import mlflow
+
+mlflow.set_tracking_uri("http://localhost:5000")
+mlflow.set_registry_uri("uc:http://localhost:8080")
```
Comment (kriscon-db) [inline docs/quickstart.md:184]: The tracking uri does not seem to like localhost.  I will change both to be 127.0.0.1 as that works for both.

File: docs/usage/models.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+This page shows you how to use Unity Catalog to store, access and govern Registered Models and Model Versions..
+
+Registered Models are logical containers for ML models within Unity Catalog.  A registered model is comprised of any
+number of Model Versions which represent different iterations of the model that you with to keep track of from within Unity Catalog.
+
+Unity catalog provides all of the functionality needed to keep track of model in your model development workflow.
+
+The following diagram shows an example of a Unity Catalog instance with a registered model: `iris_regression` that contains 1 model version:
+
+![UC Functions](../assets/images/uc_functions.png)
```
Comment (kriscon-db) [inline docs/usage/models.md:12]: @dennyglee @MrPowers We need a similar image created for this link to show how registered models and model versions work in UC.

**Pull request merged**